tests/test_brownian_interval.py

```diff
@@ -35,8 +36,9 @@
 SMALL_BATCH_SIZE = 16
 LARGE_BATCH_SIZE = 131072
 REPS = 3
+MEDIUM_REPS = 25
 LARGE_REPS = 500
-ALPHA = 0.001
+ALPHA = 0.0001
```

Comment:
We're now making lots and lots of tests - enough that we trigger this p-value at least once.
When I've had a bug and got a true positive from the test, then I've usually noticed very small p-values of 10^-180 etc, so this didn't seem likely to give any false negatives.

Reply:
Makes sense. Do you know how long it takes to run the new tests with `MEDIUM_REPS` on a typical machine disabling CUDA?  

I'm asking since long tests could slow down future dev cycles.

Reply:
I just timed it. 3 minutes in total. Not quite the zippy few seconds it was before, but not prohibitive IMO.

Reply:
I see. Is there a particular advantage in doing 25 times the inner loop? Would there be a problem if just did `REPS=3` for it?

Reply:
It's to be more sure that the complicated dependency structure of BInterval doesn't break anything. So I'd prefer to leave it as-is, but that's not a super strong preference if you really want to remove it.

Reply:
I see. That makes sense, since the query times and seeds are stored. Let's keep it like that then.